djangocms_bootstrap4/contrib/bootstrap4_grid/templates/djangocms_bootstrap4/grid_row.html

```diff
@@ -0,0 +1,7 @@
+{% load cms_tags %}
+
+<{{ instance.tag_type }} {{ instance.attributes_str }} style="padding: 20px; border: 4px dashed #333;">
```

Comment:
same

Reply:
Right, will remove them, is for testing and demoing :)